app/socket-apis/__tests__/complete-round.js

```diff
@@ -0,0 +1,112 @@
+// https://github.com/EnCiv/civil-pursuit/issues/212
+const dturn = require('../../dturn/dturn')
+const completeRound = require('../complete-round')
+const { insertStatementId, getStatementIds } = dturn
+
+const userId = '12345678abcdefgh'
+const synuser = { synuser: { id: userId } }
+
+const UInfoHistory = []
+
+beforeEach(async () => {
+  jest.spyOn(console, 'error').mockImplementation(() => {})
+  UInfoHistory.length = 0
+})
+
+afterEach(() => {
+  console.error.mockRestore()
+  jest.restoreAllMocks()
+})
+
+// Test 1: User not logged in
+test('Return undefined if user is not logged in.', async () => {
+  await dturn.initDiscussion('discussion1', {
```

Comment:
For all id's like discussionId, statementId, userId, please use a string that contains only 0-9||a-f  so that it's more like a valid objectId.